packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -354,7 +394,7 @@ const Cart = () => {
                                     {basket.productItems?.map((productItem, idx) => {
                                         return (
                                             <ProductItem
-                                                key={productItem.productId}
+                                                key={`${productItem.productId}-${idx}`}
```

Comment:
We can no longer rely on the basket items having distinct product ids. With product bundle, it's now possible to have 2 product bundles appearing as 2 separate line items, while they both have the same product ids.

Reply:
We need to use `itemId` or `UUID` instead

Reply:
We could also use variant IDs from the children, however, with the way bundles is currently implemented, there would still potential collisions as you can have multiple bundles with the same child variant selections existing as multiple line items